README/ReleaseNotes/v620/index.md

```diff
@@ -121,6 +121,23 @@ or in `TBrowser` by opening `Browser Help → About ROOT`.
 * Avoid deleted memory access in `MakeProject` and in handling of
  `I/O customization rules`.
 
+### Compression algorithms
+
+* Added a new compression algorithm ZSTD, a dictionary-type algorithm (LZ77) with large search window and fast implementations of entropy coding stage, using either very fast Finite State Entropy (tANS) or Huffman coding [https://github.com/facebook/zstd.git]. ZSTD offers a better compression ratio, and faster decompression speed: 2x faster then ZLIB and 6x faster comparing to LZMA (file compressed with ZSTD is only ~20 % bigger comparing to LZMA). ZSTD provides a wide range of compression levels, and after evaluation we recommend to use for your purposes compression level 5 (or 6). 
```

Comment:
Consider moving the zstd link to right after "compression algorithm ZSTD", else it looks like a link for Huffman coding.

Reply:
Done!